components/forms/TextField.vue

```diff
@@ -3,6 +3,9 @@
 const props = defineProps<TextFieldProps & {
   icon?: string
   loading?: boolean
+  trailing?: boolean
```

Comment:
po co ten prop?

Reply:
to nie jest odpowiedz

Reply:
przeczytaj do cholery co ten prop robi na tych docsach i wytlumacz mi po co go dodales

Reply:
A, uznalem ze skoro jest taki prop ogolnie i mielismy go chyba wczesniej w komponencie to i tak powinien zostac, na tym polegalo to ze wyslalem tylko obrazek. Jednak masz racje, gdy mamy podzial na leading-icon i trailing-icon, ten prop jest zbedny podobnie jak prop icon

Reply:
Bo to czy jest on na true lub false, nie robi roznicy